presqt/management/commands/validate_target_json.py

```diff
@@ -20,6 +20,9 @@ def handle(self, *args, **options):
         # If JSON Schema validation has failed
         if validation is not True:
             print(validation)
+            print(
```

Comment:
Looks like you've got some duplicate code here and below... perhaps pull out into a little function?